packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -539,64 +549,64 @@ class AnswerOption extends React.Component<
             </Button>
         );
 
-        const answerStatusCss = css(
-            styles.answerStatus,
-            this.props.considered === "wrong" && styles.answerStatusWrong,
-            this.props.considered === "correct" && styles.answerStatusCorrect,
-            this.props.considered === "ungraded" && styles.answerStatusUngraded,
-        );
-
         return (
             <div className={css(styles.answerOption)}>
-                <div className={css(styles.answerBody)}>
-                    <div>
-                        <button
-                            onClick={this.toggleConsidered}
-                            className={answerStatusCss}
-                        >
-                            {this.props.considered}
-                        </button>
-
-                        <div>
-                            <Expression {...this.props.expressionProps} />
-                        </div>
-                    </div>
-
-                    <div className={css(styles.paddedY, styles.paddedX)}>
-                        <Checkbox
-                            label="Answer expression must have the same form."
-                            checked={this.props.form}
-                            onChange={this.props.onChangeForm}
-                        />
-                        <InfoTip>
-                            <p>
-                                The student&apos;s answer must be in the same
-                                form. Commutativity and excess negative signs
-                                are ignored.
-                            </p>
-                        </InfoTip>
-                    </div>
-
-                    <div className={css(styles.paddedY, styles.paddedX)}>
-                        <Checkbox
-                            label="Answer expression must be fully expanded and simplified."
-                            checked={this.props.simplify}
-                            onChange={this.props.onChangeSimplify}
-                        />
-                        <InfoTip>
-                            <p>
-                                The student&apos;s answer must be fully expanded
-                                and simplified. Answering this equation
-                                (x^2+2x+1) with this factored equation (x+1)^2
-                                will render this response &quot;Your answer is
-                                not fully expanded and simplified.&quot;
-                            </p>
-                        </InfoTip>
-                    </div>
-
-                    <div className={css(styles.buttonRow, styles.paddedY)}>
-                        {removeButton}
-                    </div>
+                <ButtonGroup
```

Comment:
This is the core of the change. Instead of a coloured div that you have a to know to click on to change, we present a button group where all options are visible and the selected option is highlighted! 

<img width="212" alt="image" src="https://github.com/user-attachments/assets/993f979d-611a-4952-aa79-f3996b176552" />